CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Client with Java 8 runtime and Apache HttpClient 5 Transport fails with java.lang.NoSuchMethodError: java.nio.ByteBuffer.flip()Ljava/nio/ByteBuffer ([#13100](https://github.com/opensearch-project/opensearch-java/pull/13100))
 - Fix implement mark() and markSupported() in class FilterStreamInput ([#13098](https://github.com/opensearch-project/OpenSearch/pull/13098))
 - Fix snapshot _status API to return correct status for partial snapshots ([#12812](https://github.com/opensearch-project/OpenSearch/pull/12812))
+- Fix mapper_parsing_exception when using flat_object fields with names longer than 11 characters, which previously led to unexpected errors during index operations ([#13259](https://github.com/opensearch-project/OpenSearch/pull/13259))
```

Comment:
You can remove ", which previously led to unexpected errors during index operations" that's basically the same thing as "mapper_parsing_exception"